test/session_test.dart

```diff
@@ -120,7 +120,7 @@ Future<void> main([List<String>? args]) async {
   });
 
   baasTest('SyncSession.waitForUpload with no changes', (configuration) async {
-    final realm = await getIntegrationRealm();
+    final realm = await getIntegrationRealm(configuration);
```

Comment:
Why do we need to pass the app config as required now.

Reply:
Not needed.